read-fonts/src/tests/test_data.rs

```diff
@@ -329,3 +329,20 @@ pub mod post {
         0x4, 0x68, 0x6f, 0x6c, 0x61, // 4, h o l a
     ]);
 }
+
+pub mod cpal {
+    use crate::{tables::cpal::Cpal, FontRef, TableProvider};
+
+    use super::test_fonts::COLR_GRADIENT_RECT;
+
+    pub fn sample() -> Cpal<'static> {
+        return FontRef::new(COLR_GRADIENT_RECT).unwrap().cpal().unwrap();
+    }
+}
+
+pub mod test_fonts {
+    use crate::FontData;
+
+    pub static COLR_GRADIENT_RECT: FontData<'static> =
+        FontData::new(include_bytes!("test_fonts/linear_gradient_rect_colr_1.ttf"));
```

Comment:
I would keep test data out of the `src` directory; maybe `resources/test_fonts`?

(including these in the src directory will cause them to be automatically included in the `.crate` that is uploaded to crates.io/downloaded by any users)

Reply:
TIL! Will fix.

Reply:
Fixed